recipes/pudb/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set version = "2016.2" %}
+
+package:
+  name: pudb
+  version: {{ version }}
+
+source:
+  fn: pudb-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/p/pudb/pudb-{{ version }}.tar.gz
+  md5: 4573b70163329c1cb59836a357bfdf7c
+
+build:
+  # conda skeleton added this, but I'm unsure why it would be needed.
+  # preserve_egg_dir: True
+  entry_points:
+    - pudb = pudb.run:main
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  # Urwid is skipped on Windows
+  skip: true  # [win]
+requirements:
+  build:
+    - python
+    - setuptools
+    - urwid >=1.1.1
+    - pygments >=1.0
+  run:
+    - python
+    - setuptools
+    - urwid >=1.1.1
+    - pygments >=1.0
+
+test:
+  imports:
+    - pudb
+
+  commands:
+    - pudb --help
+
+about:
+  home: http://pypi.python.org/pypi/pudb
+  license: MIT License
+  summary: 'A full-screen, console-based Python debugger'
+
+  dev_url: https://github.com/inducer/pudb
+
+extra:
+  recipe-maintainers:
+    - asmeurer
```

Comment:
PuDB installs `pudb3` in Python 3. I changed it in this recipe to always install `pudb`.